testservers/setup.go

```diff
@@ -22,7 +20,7 @@ const (
 const (
 	red = 31 + iota
 	green
-	yellow
+	_ // yellow
```

Comment:
Is this necessary?

Reply:
You can leave `yellow` but it isn't used anywhere. The `_` preserves the correct numbering without an unused symbol.